javascript/javascript2/week3/Game/game.js

```diff
@@ -0,0 +1,89 @@
+//When the button is clicked, get the value of the input. This value will be the amount of time the game should run.
+let startBtn = false;
+let counterS = 0;
+let counterL = 0;
+
+let confettiS;
+let confetti;
+
+const listenForClick = function (event) {
+  const secSetup = document.getElementById("secSetup").value;
+
+  if (secSetup === "" || secSetup <= 0) {
+    alert("fill up the time-window. it has to be more than 1");
+    return;
+  }
+
+  let durationOfOneGame = secSetup * 1000;
+  document.getElementById("winnerS").innerText = "";
+  document.getElementById("winnerL").innerText = "";
+
+  if (confettiS) {
+    confettiS.clear();
+  }
+
+  if (confetti) {
+    confetti.clear();
+  }
+
+  let confettiSettingsS = { target: "my-canvasS" };
+  confettiS = new ConfettiGenerator(confettiSettingsS);
+  let confettiSettings = { target: "my-canvasL" };
+  confetti = new ConfettiGenerator(confettiSettings);
+
+  if (event.target.id === "startBtn") {
+    startBtn = true;
+  }
+
+  setTimeout(() => {
+    startBtn = false;
+    alert("the game is over");
```

Comment:
`alert` is very old fashioned javascript, try to use the DOM to give information to the user :)